source/priority_queue.c

```diff
@@ -159,10 +176,11 @@ void aws_priority_queue_init_static(
     AWS_ZERO_STRUCT(queue->backpointers);
 
     aws_array_list_init_static(&queue->container, heap, item_count, item_size);
+
+    AWS_POSTCONDITION(aws_priority_queue_is_valid(queue));
 }
 
 bool aws_priority_queue_is_valid(const struct aws_priority_queue *const queue) {
-
```

Comment:
Unrelated change?

Reply:
Should I separate them from this PR? I added some postconditions to two other functions in `priority_queue.c`.

Reply:
I'd argue to keep the pre and postconditions along with the proof in the same PR.